docs/resources/howto/wiki.md

```diff
@@ -0,0 +1,273 @@
+# Editing Wiki pages
+
+This page presents an overview of the steps necessary to modify the wiki and add new pages.
+It also contains examples for useful commands and can be used as a template for new pages.
+
+## Setup
+
+### To Make Changes Online
+
+This is a method adapted to making small changes, most likely to a single page.
+Go to the current version of the site [here][site_address], navigate to a page you wish to modify and click the page's modification link (the little pen at the top right).
+You will be taken to the Github GUI to make your changes, which you can later commit.
+
```

Comment:
!!! note
    As a lot of fancy styling is added by the [material theme][mkdocs_material] used, only the basic markdown formatting is seen in the preview.


[mkdocs_material]: https://squidfunk.github.io/mkdocs-material/

Reply:
I am not a huge fan of large changes to this section, since it's the same as the README. So IMHO this should then also be added to the README, to stay consistent (if we want consistency between these parts).